.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Scope 
+Description of feature / issue, and links to issue...
```

Comment:
@ShaunEgan I tend to agree, I also think in time as there is more consistency throughout the team these things could fall away